dir.props

```diff
@@ -10,7 +10,7 @@
 
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00089</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00091</BuildToolsVersion>
```

Comment:
FYI - I'm going to need to publish a new buildtools package today and update it so it may conflict with this change depending one when you get yours merged.